### ISSUE TITLE
class Configuration vs driver.config

### ISSUE THREAD

**Issue by yashaka** [KnowledgeExpert/selenidejs#17]
why different? what about consistency?

why not class Configuration & driver.configuration or class Config & driver.config?

**Comment by AleksanderPopov** [comment]
@yashaka yes, agree this is the problem...it was because of `Browser.setDriver` function, where `Browser` namespace keeps `config` variable, and if that variable would be `configuration`, it will cause collision with `Browser.setDriver` `configuration?: Configuration` parameter....

Any ideas how we can get rid of it?
